src/Microsoft.Extensions.Configuration.AzureAppConfiguration/Constants/RequestTracingConstants.cs

```diff
@@ -17,6 +21,9 @@ internal class RequestTracingConstants
 
         public const string RequestTypeKey = "RequestType";
         public const string HostTypeKey = "Host";
+        public const string DevEnvironmentTag = "DevEnvironment";
+        public const string KvrConfiguredTag = "KeyVaultConfigured";
```

Comment:
Maybe just `KvConfigured` to match the constant.

Reply:
In that case, I think `KeyVaultConfigured` is more descriptive and makes more sense in logs than `KvrConfiguredTag`.
How about this:
` public const string KeyVaultConfiguredTag = "KeyVaultConfigured";`